src/pages/en/guides/markdown-content.md

```diff
@@ -146,8 +44,10 @@ const { content } = Astro.props;
   </body>
 </html>
 ```
+### The content prop
+For Markdown files, the `content` prop also has an `astro` property which holds special metadata about the page such as the complete Markdown `source` and a `headers` object. 
```

Comment:
```suggestion

The `content` prop, contains all your frontmatter properties that are defined onto the page. It also holds special prop named `astro` which contains additional metadata about the page such as the complete Markdown `source` and a `headers` object. 
```

Reply:
I do like starting the phrase with "The content prop.." so I'll amend.